eng/config/PublishData.json

```diff
@@ -120,16 +120,14 @@
     },
     "dev15.9.x-vs-deps": {
       "nugetKind": "PerBuildPreRelease",
-      "vsBranch": "rel/d15.9",
-      "vsMajorVersion": 15
```

Comment:
I think our official builds always read this file from the main branch, do we need to do some manual testing to be sure this won't break servicing branch builds? We have a monthly smoke test that we are supposed to keep an eye on, FWIW.

Reply:
I put my local enlistment into a time machine and we don't even use vsMajorVersion in our 15.9 code

Reply:
In this case `rel/d15.9` isn't even a real branch, we use `dev15.9.x` so think this entire section is suspect.